ifrs17/Import/ImportStorage.ipynb

```diff
@@ -367,8 +367,30 @@
                 "\n        return Math.Pow(1d + rate[period].Values[0], 1d / 12d) - 1d;",
                 "\n    }",
                 "\n    ",
-                "\n    public double GetPremiumAllocationFactor(ImportIdentity id) => SingleDataNodeParametersByGoc.TryGetValue(id.DataNode, out var singleDataNodeParameter) ",
-                "\n                                                                                ? singleDataNodeParameter[CurrentPeriod].PremiumAllocation : DefaultPremiumExperienceAdjustmentFactor;",
+                "\n    public (string, double[]) GetReleasePattern (ImportIdentity identity, string amountType, int patternShift)",
+                "\n    {",
+                "\n        var patternFromCashflow = GetValues(identity with {AocType = AocTypes.CL, Novelty = Novelties.C}, amountType, EstimateTypes.P, (int?)null);",
+                "\n        if (patternFromCashflow.Any())",
+                "\n            return (amountType, Enumerable.Repeat(0d, patternShift).Concat(patternFromCashflow).ToArray());",
```

Comment:
so if I understand it correctly, the patternFromCashflow is not to be interpolated etc., it will be returned as is. correct?

Reply:
yes, CF is already monthly